Project.toml

```diff
@@ -13,6 +13,7 @@ Distances = "b4f34e82-e78d-54a5-968a-f98e89d6e8f7"
 DocStringExtensions = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
 FFTW = "7a1cc6ca-52ef-59f5-83cd-3a7055c09341"
 GPUArrays = "0c68f7d7-f131-5f86-a1c3-88cf8149b2d7"
+GPUCompiler = "61eb1bfa-7361-4325-ad38-22787b887f55"
```

Comment:
This is fixed in Enzyme now, no need to involve GPUCompiler anymore.

Reply:
looks like tests are still failing, but may be we need to wait a few more min?

Reply:
Where are you looking at?  I only see https://github.com/CliMA/Oceananigans.jl/actions/runs/14063874231/job/39381278166?pr=4272#step:6:284, which is a problem in `NCDatasets`, unrelated to Enzyme/Reactant/GPUCompiler

Reply:
Ah, https://github.com/CliMA/Oceananigans.jl/actions/runs/14063874231/job/39381277641?pr=4272#step:6:288 just failed, that's a bug in Reactant